src/app/wishlists/wishlist-fetch.ts

```diff
@@ -4,19 +4,37 @@ import _ from 'lodash';
 import { showNotification } from 'app/notifications/notifications';
 import { loadWishLists } from './actions';
 import { ThunkResult } from 'app/store/reducers';
+import { setSetting } from 'app/settings/actions';
 
-export function fetchWishList(): ThunkResult<Promise<void>> {
+function hoursAgo(dateToCheck?: Date): number {
+  if (!dateToCheck) {
+    return 99999;
+  }
+
+  return Math.abs(new Date().getTime() - dateToCheck.getTime()) / (1000 * 60 * 60);
+}
+
+export function fetchWishList(ignoreThrottle: boolean): ThunkResult<Promise<void>> {
   return async (dispatch, getState) => {
     const wishListSource = getState().settings.wishListSource;
 
     if (!wishListSource) {
       return;
     }
 
+    const wishListLastUpdated = getState().settings.wishListLastUpdated;
+
+    if (!ignoreThrottle && hoursAgo(wishListLastUpdated) < 24) {
+      return;
+    }
+
     const wishListResponse = await fetch(wishListSource);
     const wishListText = await wishListResponse.text();
 
     dispatch(transformAndStoreWishList(wishListText, 'Fetch Wish List'));
+
+    const currentDate = new Date();
+    dispatch(setSetting('wishListLastUpdated', currentDate));
```

Comment:
This is the time the wishlist was last *fetched*, then - not the last time the setting was changed.

1. I'd much prefer if this was stored locally as part of the wish list state, rather than in settings. Settings get copied between machines, but the actual wish list contents don't. This describes locally cached data, so it should be saved along with that local cache, not in settings.
2. If it *were* saved in settings, you'd want this to be part of the reducer implementation for `transformAndStoreWishList`, not a separate action that gets dispatched afterwards.